src/usr/local/www/vpn_ipsec_settings.php

```diff
@@ -152,6 +157,19 @@
 			unset($config['ipsec']['acceptunencryptedmainmode']);
 		}
 
+		if (isset($_POST['maxexchange']) && (strlen($_POST['maxexchange']) > 0)) {
+			if (!isset($config['ipsec']['maxexchange'])) {
+				$needsrestart = true;
+			} elseif ($pconfig['maxexchange'] != $config['ipsec']['maxexchange']) {
+				$needsrestart = true;
+			}
```

Comment:
This statement could be simplified using `||` instead of `elseif`

Reply:
Fix already added, something else I can do?